package.json

```diff
@@ -90,7 +91,6 @@
     "espree": "^10.0.0",
     "estraverse": "^5.3.0",
     "rollup": "^4.9.6",
-    "typescript": "^5.3.3",
```

Comment:
I don't want to rely on a global package.

Reply:
This is more of a `pnpm` thing as I use vanilla `npm` where peer dependencies are automatically installed at the top level. `esm-d-ts` has a peer dependency for the latest `typescript`. `pnpm` without a `.npmrc` file with `shamefully-hoist=true` does not make `tsc` available by default. The "shamefully" part of that option name is probably more humorous because of the design goal of `pnpm` isolating the dependency tree more than other package managers.

Using `pnpm` one can explicitly include `typescript` or go the  `shamefully-hoist=true` configuration route. There is no problem in adding back `typescript` as a `devDependency` in this case. That specific TS version will also be used by `esm-d-ts`.